styles/_footer.scss

```
@@ -57,67 +133,77 @@
       display: block;
       margin: 0;
       font-style: normal;
+      white-space: pre-line;
+    }
+
+    .copyright {
+      text-align: left;
     }
   }
 
-  .footer-nav {
-    margin: 0;
-    padding: 0;
-    list-style: none;
+  .right-info {
     text-align: center;
+    padding: 0 0 20px;
+    // TODO: remove that line while adding hubspot form in the footer
+    display: block;
 
-    @include tablet {
+    @include desktop {
       float: right;
+      text-align: left;
+      padding: 0;
+    }
+  }
+
+  .footer-nav {
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    @include desktop {
+      float: left;
     }
 
     li {
       display: inline-block;
       vertical-align: top;
-      font-size: 20px;
+      margin: 0 0 0 10px;
+      font-size: 18px;
       line-height: 28px;
-      margin: 0 15px;
 
-      @include tablet {
-        margin: 0 0 0 30px;
+      @include desktop {
+        margin: 0 0 0 15px;
       }
 
-      a {
-        color: white;
+      &:first-child {
+        margin: 0;
       }
     }
   }
 
-  .center-content {
-    text-align: center;
-    overflow: hidden;
+  .footer-sub-nav {
+    margin: 0 0 5px;
+    list-style: none;
+    text-align: left;
+    font-size: 15px;
```

This should be `14px` as per invision design. The company info on the left of the footer links is already 14.
```suggestion
    font-size: 14px;
```